packages/blockchain-api/src/blockscout.ts

```diff
@@ -149,12 +150,13 @@ export class BlockscoutAPI extends RESTDataSource {
 
   async getTokenTransactions(args: TokenTransactionArgs) {
     const userAddress = args.address.toLowerCase()
-    const token = args.token
+    const { token, tokens: receivedTokens } = args
```

Comment:
Made this change so that the app can send the tokens they want to see. We still need blockscout to return cEUR transactions for them to show up on Valora.

Reply:
Blockscout is already returning cEUR transactions in alfajores3, I deployed blockchain-api with these changes, so this is live now in Valora alfajores.